_data/rules.yml

```diff
@@ -1090,6 +1090,29 @@ groups:
           - name: cloudflare/kafka_zookeeper_exporter
             doc_url: https://github.com/cloudflare/kafka_zookeeper_exporter
             rules:
+          - name:  dabealu/zookeeper-exporter
+            doc_url: https://github.com/dabealu/zookeeper-exporter
+            rules:
+              - name: Zookeeper Down
+                description: "Zookeeper down on instance {{ $labels.instance }}"
+                query: 'zk_up == 0'
+                severity: critical
+                for: 3m
```

Comment:
You can remove the `for` clause. It will fire immediately.

Reply:
Hmm, in my opinion, immediate alert firing can lead to some false alarms and alert fatigue. Maybe just decrease it to `1m` to handle planned Zk restarts, for example?

Reply:
In that case, we are not alerting for a container crash, but for a critical infrastructure component.

If ZK is not able to restart by itself ASAP, I think you can ping Ops.

WDYT ?

Reply:
It depends on real scenarios (e.g., in our installation, losing one Zk node is not crucial), but your argument has good reasons. Anyway, it is just examples, and they should be adapted for real cases. :)

Reply:
fixed